gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -205,7 +205,11 @@ const CssVariablesPage = () => {
             </NxWarningAlert>
           </PropertyDocItem>
           <PropertyDocItem propertyVar="--nx-border-width">
-            Standard border-width value used across multiple elements.
+            The standard border-width value.
+          </PropertyDocItem>
+          <PropertyDocItem propertyVar="--nx-border-width-alt">
+            An alternative border-width value that is identical to <NxCode>--nx-border-width</NxCode> in light mode,
+            but 1px thicker in dark mode.
```

Comment:
I feel like this is a good balance of meaningfully expressing what it does without locking ourselves into too many specifics.

```suggestion
            but thicker in dark mode.
```